src/openvtb/components/taken/tests/test_json_schemas.py

```diff
@@ -0,0 +1,434 @@
+                self.externe_taak.save()
+            self.assertEqual(
+                error.exception.message_dict,
+                {"data": ["'bedrag' is a required property"]},
+            )
+
+        with self.subTest("'valuta' field required"):
+            self.externe_taak.data = {
+                "bedrag": 10.12,
+                # "valuta": "EUR",
+                "transactieomschrijving": "test",
+                "doelrekening": {
+                    "naam": "test",
+                    "iban": "iban-code-test",
+                },
+            }
+            with self.assertRaises(ValidationError) as error:
+                self.externe_taak.full_clean()
+                self.externe_taak.save()
+            self.assertEqual(
+                error.exception.message_dict,
+                {"data": ["'valuta' is a required property"]},
+            )
+
+        with self.subTest("'transactieomschrijving' field required"):
+            self.externe_taak.data = {
+                "bedrag": 10.12,
+                "valuta": "EUR",
+                # "transactieomschrijving": "test",
+                "doelrekening": {
+                    "naam": "test",
+                    "iban": "iban-code-test",
+                },
+            }
+            with self.assertRaises(ValidationError) as error:
+                self.externe_taak.full_clean()
+                self.externe_taak.save()
+            self.assertEqual(
+                error.exception.message_dict,
+                {"data": ["'transactieomschrijving' is a required property"]},
+            )
+
+        with self.subTest("'doelrekening' field required"):
+            self.externe_taak.data = {
+                "bedrag": 10.12,
+                "valuta": "EUR",
+                "transactieomschrijving": "test",
+                # "doelrekening": {
+                # "naam": "test",
+                # "iban": "iban-code-test",
+                # },
+            }
+            with self.assertRaises(ValidationError) as error:
+                self.externe_taak.full_clean()
+                self.externe_taak.save()
+            self.assertEqual(
+                error.exception.message_dict,
+                {"data": ["'doelrekening' is a required property"]},
+            )
+
+        with self.subTest("'doelrekening.naam' field required"):
+            self.externe_taak.data = {
+                "bedrag": 10.12,
+                "valuta": "EUR",
+                "transactieomschrijving": "test",
+                "doelrekening": {
+                    # "naam": "test",
+                    "iban": "iban-code-test",
+                },
+            }
+            with self.assertRaises(ValidationError) as error:
+                self.externe_taak.full_clean()
+                self.externe_taak.save()
+            self.assertEqual(
+                error.exception.message_dict,
+                {"data": ["'naam' is a required property"]},
```

Comment:
I don't know if this is possible with the jsonschema library, but it would be nice if this somehow put the full path here

```suggestion
                {"data": ["'doelrekening.naam' is a required property"]},
```